### DISCUSSION TITLE
Rack Request add upload file "doc"

### DISCUSSION THREAD

**Discussion by alex001x** [opendcim/openDCIM#1574] [Ideas]
Hi,

it would be nice to add a feature to the rack request form "add attachment" for technical documentation

Alex.

**Comment by wilpig** [comment]
Probably gonna say no on this, we've restricted the file types in an attempt to prevent malware abuse scenarios. We could possibly add doc, docx, PDF but we'd really need to discuss especially with all the abuse those formats have had over the years

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: Alexandre ***@***.***>
Sent: Wednesday, October 30, 2024 10:05:04 AM
To: opendcim/openDCIM ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [opendcim/openDCIM] Rack Request add upload file "doc" (Discussion #1574)


Hi,

it would be nice to add a feature to the rack request form "add attachment" for technical documentation

Alex.

—
Reply to this email directly, view it on GitHub<https://github.com/opendcim/openDCIM/discussions/1574>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AAMTY6U3JWEBOATB743CH43Z6EGUBAVCNFSM6AAAAABQ4SUTR2VHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZXGM4TOMRXG4>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/opendcim/openDCIM/discussions/1574", "url": "https://github.com/opendcim/openDCIM/discussions/1574", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]
